test/DynamoCoreTests/SchedulerTests.cs

```diff
@@ -1303,7 +1314,9 @@ private AsyncTask MakeSetTraceDataAsyncTask()
 
         private AsyncTask MakeUpdateGraphAsyncTask()
         {
-            return new FakeUpdateGraphAsyncTask(MakeAsyncTaskData());
+            var t = new FakeUpdateGraphAsyncTask(MakeAsyncTaskData());
+            t.InitializeTestData(); // Just to initialize ModifiedNodes
+            return t;
```

Comment:
Adding a call to `InitializeTestData` just to initialize `ModifiedNodes` list so that it will never be `null`.